apps/web/src/components/Feed/FeedEventItem.tsx

```diff
@@ -110,6 +115,18 @@ export default function FeedEventItemWithBoundary({
     onPotentialLayoutShift(index);
   }, [index, onPotentialLayoutShift]);
 
+  // check that subEvents is non-empty
+  if (event.eventData?.__typename !== 'UserFollowedUsersFeedEventData') {
+    // ignore GalleryInfoUpdatedFeedEventData events because we have no component to handle that right now
+    const subEvents = event?.eventData?.subEventDatas
+      ?.slice(0, 4)
+      .filter((event) => event.__typename !== 'GalleryInfoUpdatedFeedEventData');
+
+    if (!subEvents?.length) {
+      return null;
+    }
+  }
+
```

Comment:
kind of a hacky fix but basically was trying to ignore the eventDataType`GalleryInfoUpdatedFeedEventData` because we don't have a component for it from this file so it would always be an empty item for this event type. I wanted to know if you had a better idea of how I could fix this @Robinnnnn! thanks

<img width="243" alt="Screenshot 2024-03-23 at 15 43 19" src="https://github.com/gallery-so/gallery/assets/49758803/13e42f02-f59d-43fb-9651-87cb817d256c">

Reply:
sorry but why can't we just introduce a new component in the screenshot you provided?